pkgs/by-name/ka/kata-runtime/package.nix

```
@@ -0,0 +1,104 @@
+# Derived from https://github.com/colemickens/nixpkgs-kubernetes
+{
+  lib,
```

```suggestion
  lib,
```
The case for alphabetical ordering:

Any newcomer gets alphabetical ordering right *every time*.
Having some 'keywords' at beginning breaks alphabetical order and leads people to not following alphabetical order at all.
There is no consensus on which keywords should come first if not following alphabetical order.
Alphabetical order helps finding duplicated arguments early. It also facilitates look-up.
To me, alphabetical order is the best criteria we can have, given that everyone knows it already, and no arbitrary criteria is necessary for order of parameters. And it can be easily automated.
My goal in making this comment is such that you also get to see the beauty of it.
I think, we should standardize in alphabetical order (eventually).

---

This came from `nixfmt`.

---

Oh I see, `nixfmt` doesn't actually sort these. Okay, I'll change it.

---

I hope one day `nixfmt` sorts for alphabetical order. Instead of another arbitrary criteria that every newcomer will get wrong. (Since I do a lot of reviews, I don't want to do more work nudging people on the right direction on this.)

---

> Oh I see, `nixfmt` doesn't actually sort these. Okay, I'll change it.

Goal is that `nixfmt` will one day hopefully automatically sort it. But then, there will be a lot of files in conflict. I'm just voting with code here. And trying to spread the alphabetical order mind virus. ;-)

---

Works for me!